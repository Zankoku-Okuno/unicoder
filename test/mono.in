\lambda
\lambda x
\lambda.x
\lambdax
\lambda{x}
