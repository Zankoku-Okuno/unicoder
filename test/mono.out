位
位 x
位x
\lambdax
位{x}
